test/main.js

```diff
@@ -24,6 +26,8 @@ var SelfInsertingView = View.extend({
     render: function () {}
 });
 
+viewCompliance.view(test, ViewSwitcher, { view: new makeTestView()() });
```

Comment:
> Can regular, non-ampersand objects achieve ampersand-view-compliance?

Yes, any object implementing `render` and `remove` methods can be used.

> In the meantime, would it be worth having ViewSwitcher extend AmpersandView?

This module is so small, that I believe introducing any dependencies is not necessary.